components/OutcomeModal.vue

```diff
@@ -327,6 +328,9 @@ export default {
   },
   methods: {
     async submit() {
+      if (this.type === 'Taken' && !this.tookUsers.length)
+        return (this.submittedWithNoSelectedUser = true)
```

Comment:
Should we reset `submittedWithNoSelectedUser ` to false?  If you go through this code once, then come through again but fail to submit because of `submitDisabled` then `submittedWithNoSelectedUser ` would still be true.